### PR TITLE
Reproducible build - sorted javax.inject.Named

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -179,6 +179,13 @@
                 <artifactId>plexus-utils</artifactId>
                 <version>3.3.0</version>
             </dependency>
+
+            <dependency>
```
Comment (mkarg) [inline pom.xml:183]: Does it make sense to file a RFE with the maven core tracker? It seems to problem is simply induced by maven core, and not really originates from the maven sign plugin.

**Review by mkarg — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (slawekjaranowski) [reply pom.xml:183]: It was updated for maven 3.6.3 https://github.com/apache/maven/commit/bd10f00b68eb766cd239275e8504acedc2818d69
We depends on maven 3.6.0 api it is enough for us.

Comment (mkarg) [reply pom.xml:183]: Maven 3.6.3 was released more than a year ago. Why don't we simply step up? Why should someone have a problem with migrating from 3.6.0 to 3.6.3?

Comment (slawekjaranowski) [reply pom.xml:183]: I know from my experience that there are environments where updates is not enough quickly occurs. :-(
If we really needn't newer version I prefer to stay with 3.6.0

Comment (mkarg) [reply pom.xml:183]: Well, we talk about a dot-dot-increment by three steps here only...

**Pull request merged**